2 Business layer/CandidateEvaluator.Core/Handlers/Commands/CreateCategoryHandler.cs

```diff
@@ -0,0 +1,32 @@
+﻿using CandidateEvaluator.Contract.Commands.Category;
+using CandidateEvaluator.Contract.Handlers;
+using CandidateEvaluator.Contract.Models;
+using CandidateEvaluator.Contract.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CandidateEvaluator.Core.Handlers.Commands
+{
+    public class CreateCategoryHandler : ICommandHandler<CreateCategory>
+    {
+        private readonly ICategoryService _categoryService;
+
+        public CreateCategoryHandler(ICategoryService categoryService)
+        {
+            _categoryService = categoryService;
```

Comment:
I believe that handlers should operate straight on repositories/context. The service layer is redundant here.